.ci/doc/templates/catch.tpl.java

```diff
@@ -0,0 +1,25 @@
+import io.kuzzle.sdk.Kuzzle;
```

Comment:
I didn't get the purpose of this snippet template

Reply:
Used by the `createMyCredentials` snippet, since there is only one authentication plugin in the Kuzzle used to test, there is only one strategy so this method is supposed to throw a io.kuzzle.sdk.Exceptions.ApiErrorException